bindings/binding_core_node/src/minify.rs

```diff
@@ -16,6 +17,12 @@ use swc_core::{
 
 use crate::{get_compiler, util::try_with};
 
+#[derive(Debug, Deserialize)]
+pub enum ErrorFormatConfig {
```

Comment:
Why is this option in binding?

Reply:
The ErrorFormatConfig was introduced to provide flexibility in formatting error outputs, allowing different representations (e.g., JSON) depending on the user's needs. It was included here to streamline error handling directly within the minification process

Reply:
I mean, why is it stored in `bindings/*`? I think it's a quite general option and I believe `swc_error_reporters` is the better place to store this type.

Reply:
now I understand better thank you I will make an update